<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaPrincipal</class>
 <widget class="QMainWindow" name="VentanaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VentanaPrincipal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>461</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Datos Vehículo</string>
    </property>
    <widget class="QLineEdit" name="lineNombre">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre del vehículo	</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Número de ruedas</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineRuedas">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Potencia del motor</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="linePotencia">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo de combustible</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboCombustible">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Gasolina</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hidrogeno</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Metano</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Diésel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bio-diésel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Eléctrico</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Híbrido</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboColor">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>rojo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>plateado</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dorado</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>gris</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>grafito</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>amarillo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>marron</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>negro</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>azul</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>verde</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Matrícula</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineMatricula">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushGenerar">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>180</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Generar</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioRepuesto">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Rueda de repuesto</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioReparacion">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Kit de reparación</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Burbuja</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBurbuja">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>120</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bocinas</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBocinas">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkPortavasos">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>140</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>140</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Portavasos</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>241</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Gestión</string>
    </property>
    <widget class="QComboBox" name="comboLista">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Nuevo...</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushGuardar">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Guardar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushTipo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Total</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdTotal">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushSalir">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
